internal/jobs/stats/stats.go

```diff
@@ -68,20 +69,14 @@ func StartCollector(bufSize uint, jc types.JobConfiguration) *StatsCollector {
 		Stats:                make(map[string]map[StatType]uint),
 		WorkerVersion:        versioning.TEEWorkerVersion,
 		ApplicationVersion:   versioning.ApplicationVersion,
-		ReportedCapabilities: []types.Capability{},
+		ReportedCapabilities: teetypes.WorkerCapabilities{},
```

Comment:
[nit] This is not necessary, Go will by default automagically initialize structs to an empty struct.